Acrolinx.Sidebar/ISidebar.cs

```diff
@@ -187,12 +189,67 @@ public bool SupportCheckSelection
 
     public class CheckedEventArgs : CheckEventArgs
     {
-        internal CheckedEventArgs(string checkId, Range range) : base(checkId)
+        internal CheckedEventArgs(string checkId, Range range, JArray embedCheckInformation, string inputFormat) : base(checkId)
         {
             Range = range;
+            EmbedCheckInformation = new JObject();
+            EmbedCheckInformation["embedCheckInformation"] = embedCheckInformation;
+            if (inputFormat.ToUpper().Equals("XML"))
```

Comment:
input format might be null? It would be saver to turn it around:

    if("xml".equals(inputFormat, `StringComparison.OrdinalIgnoreCase))

See:
https://msdn.microsoft.com/de-de/library/c64xh8f9(v=vs.110).aspx